gui/mozregui/build_runner.py

```diff
@@ -154,6 +154,10 @@ def init_worker(self, fetch_config, options):
         if options.get("url") and fetch_config.app_name != "thunderbird":
             launcher_kwargs["cmdargs"] += [options["url"]]
 
+        # Lang only works for firefox-l10n
+        if options.get("lang") and options["application"] == "firefox-l10n":
```

Comment:
We should probably error out if lang is specified but the application is not `firefox-l10n`

Reply:
Good idea, done.